### PR TITLE
Paraview: Allow +shared +cuda

### PR DESCRIPTION
**Pull request by kwryankrattiger**
It seems this was added for vtk-m once, but is no longer a requirement of vtk-m and therefore can be removed from paraview.

### CODE REVIEW

**Comment by vicentebolea** [comment]
@kwryankrattiger can we hold on on this until we merge and makes its way to PV: https://gitlab.kitware.com/vtk/vtk/-/merge_requests/7864

I noticed that for the vtkm wrapper library in VTK (vtkmAccelator) there are issues with static at this moment,

**Comment by kwryankrattiger** [comment]
@vicentebolea 
> I noticed that for the vtkm wrapper library in VTK (vtkmAccelator) there are issues with static at this moment,

If there are issues with static, wouldn't it be better to merge this so we can disable static when building paraview+cuda? Or do you mean there is an issue with shared?

**Comment by vicentebolea** [comment]
@kwryankrattiger I said it in reverse :facepalm:  Yeah the issue was observed with shared libraries few months ago, but that was also a known issue in VTKm which I fixed later on, but for that we need to make some changes in the used vtkm flags in VTK (novirtuals). 

I will be back in vtkm in a couple of weeks, so how about we come  can come back to this issue then

**Comment by alalazo** [comment]
> I will be back in vtkm in a couple of weeks, so how about we come can come back to this issue then

Feel free to ping me when this is ready to be merged

**Comment by vicentebolea** [comment]
@kwryankrattiger VTK+VTKm+CUDA (with shared library) support finally made it way to VTK current master https://gitlab.kitware.com/vtk/vtk/-/merge_requests/7864

**Review by vicentebolea — CHANGES_REQUESTED** [review]

**Comment by vicentebolea** [comment]
@kwryankrattiger my apologizes, I made the review but I left it `pending for a few months without noticing.

**Comment by kwryankrattiger** [comment]
This is resovled in #28877 for VTK-m.

@vicentebolea Will ParaView 5.10.1 get this update for VTK-m as well or will that have to wait for 5.11?

@chuckatkins

**Comment by vicentebolea** [comment]
There are no plans for this, but I plan to push to move this to ParaView 5.10.1, incoming VTK release will be compatible with VTK-m 1.7

**Review by vicentebolea — APPROVED** [review]
Thanks for this!

**Comment by kwryankrattiger** [comment]
Closing this, redundant with #29054

**Pull request closed without merging**
